tests/unit/Logger.js

```diff
@@ -27,7 +27,10 @@ function filterGenerator(logLevel, callLevel) {
          * Here, patch the Config by setting a specificly designed dummyLogger
          * for testing purposes that will help us collect runtime data.
          */
-        Config.simpleLogger = dummyLogger;
+        /*
+         * Not understanding why this line is necessary, see ~line 182
```

Comment:
Well, simply not the same tests. The function here is used in the tests by the function at line 219 ( https://github.com/scality/werelogs/pull/97/files#diff-2ab315f1e2acd69edd3ade7999592af0R219 ).
Since these tests are using directly the Logger() constructor; you are not sharing the configuration instance (you're even creating independent configs for all logger modules in this commit). This means that you need to patch the configuration instance provided to each of the Loggers if you want to replace the simpleLogger for testing purposes.